docs/howtos/new_experimentdata.rst

```diff
@@ -0,0 +1,77 @@
+Instantiate a new data object for an existing experiment
+========================================================
+
+Problem
+-------
+
+You want to instantiate a new :class:`.ExperimentData` object from an existing
+experiment whose jobs have finished execution successfully.
+
+Solution
+--------
+
+.. note::
+    This guide requires :mod:`qiskit-ibm-provider`. For how to migrate from the deprecated :mod:`qiskit-ibmq-provider` to :mod:`qiskit-ibm-provider`,
+    consult the `migration guide <https://qiskit.org/documentation/partners/qiskit_ibm_provider/tutorials/Migration_Guide_from_qiskit-ibmq-provider.html>`_.\
+
+Use the code template below. You need to know the exact experiment you
+ran and its options, as well as the IDs of the jobs that were executed.
+
+.. jupyter-input::
+
+    from qiskit_experiments.framework import ExperimentData
+    from qiskit_ibm_provider import IBMProvider
+
+    # The experiment you ran
+    experiment = Experiment(**opts)
+
+    # List of job IDs for the experiment
+    job_ids= [job1, job2, ...]
+
+    provider = IBMProvider()
+
+    data = ExperimentData(experiment = experiment)
+    data.add_jobs([provider.retrieve_job(job_id) for job_id in job_ids])
+    experiment.analysis.run(data)
+
+    # Block execution of subsequent code until analysis is complete
+    data.block_for_results()
+
+``data`` will be the new experiment data object.
+
+Discussion
+----------
+
+This guide is helpful for cases such as a lost connection during experiment execution, 
+where the jobs may have finished running on the remote backends but the 
+:class:`.ExperimentData` class returned upon completion of an experiment does not 
+contain correct results.
```

Comment:
It might be worth mentioning here something like "Often you recreate the `Experiment` object by rerunning the code that you ran previously to create it. It may sometimes be helpful to save an experiment with `json.dumps(Experiment.config())` and restore it with `Experiment.from_config(json.loads(...))`.

Reply:
Good idea, added